### PR TITLE
changed the table

### PR DESCRIPTION
**Pull request by MaratMussabekov**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/8716
Sources:
https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/enable-exploit-protection
https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/exploit-protection#review-exploit-protection-events-in-windows-event-viewer

### CODE REVIEW

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9,
kindly do the copy editor review for this PR.

Dear @nam31, for your information

Thank you

**Comment by illfated** [comment]
Note to myself: the MarkDown table heading divider lines in this page are either broken or semi-broken and need repair for GitHub viewing (task for when this PR is merged: #md_cleanup ).

**Review by illfated — APPROVED** [review]
- [X] **Rich Diff view:** https://github.com/MicrosoftDocs/windows-itpro-docs/pull/8765/files?short_path=5393780#diff-5393780dd3edf2c20f5179fe0917e2ba70430d58d2efabde7d58a4d6f2185388

File: windows/security/threat-protection/microsoft-defender-atp/customize-exploit-protection.md [hunk 1]
```diff
@@ -234,20 +234,22 @@ Code integrity guard | App-level only |   BlockNonMicrosoftSigned,   AllowStoreS
...
-Validate stack integrity (StackPivot) | App-level only |   EnableRopStackPivot  | Audit not available
+Validate stack integrity (StackPivot) | App-level only |   EnableRopStackPivot  | Audit not available<a href="#r2" id="t2">\[2\]</a>
 
 <a href="#t1" id="r1">\[1\]</a>: Use the following format to enable EAF modules for dlls for a process:
 
 ```PowerShell
 Set-ProcessMitigation -Name processName.exe -Enable EnableExportAddressFilterPlus -EAFModules dllName1.dll,dllName2.dll
 ```
 
+<a href="#t2" id="r2">\[2\]</a>: Audit for this mitigation is not available via Powershell cmdlets.
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-atp/customize-exploit-protection.md:251]:
```suggestion
<a href="#t2" id="r2">\[2\]</a>: Audit for this mitigation is not available via PowerShell cmdlets.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Pull request merged**